File: codeforces/467A/go/467a.go
package main

import "fmt"

func main() {
    var n int
    fmt.Scanf("%d\n", &n)

    num := 0
    for i := 0; i < n; i++ {
        var p, q int
        fmt.Scan(&p, &q)
        if q - p >= 2 {
            num++
        }
    }

    fmt.Println(num)
}

File: codeforces/467A/cpp/467a.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, p, q, num = 0;
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> p >> q;
        if (q - p >= 2) {
            num++;
        }
    }
    cout << num << endl;
}
